wp-content/themes/humanity-theme/patterns/perspective-post-template.php

```diff
@@ -0,0 +1,67 @@
+<!-- /wp:paragraph -->
+
+<!-- wp:heading -->
+<h2 class="wp-block-heading">Add subheadings to break up the text into sections</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget placerat tellus, ac dignissim diam. Aliquam at justo vulputate, tincidunt sapien nec, efficitur erat. Aenean iaculis semper viverra. Nullam eu bibendum lacus, ut blandit risus. Curabitur nec magna sed tellus maximus mollis sit amet eget odio. Vestibulum eu arcu tellus. Praesent quam magna, maximus nec condimentum non, aliquam eu nunc. Sed urna lorem, accumsan non maximus non, maximus ut ex. Vivamus eu sem non tellus imperdiet suscipit molestie in mi. Donec vel lobortis sem. Ut pharetra ipsum a auctor eleifend.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Sed sit amet sem faucibus, elementum lacus nec, accumsan quam. Maecenas iaculis enim ac cursus facilisis. Aliquam iaculis hendrerit ex et dapibus. Nulla a ipsum vitae turpis efficitur semper. Donec accumsan mi in lacus dictum pretium. Pellentesque non urna eget lorem scelerisque vulputate. Morbi vel augue ipsum. Phasellus mi neque, efficitur id molestie rutrum, imperdiet sit amet nisi. Nulla elementum lacus at elit congue luctus. In vitae aliquet velit, a viverra turpis.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Quisque eget nibh quis sapien rutrum aliquet. Vestibulum imperdiet a nisl sodales fringilla. Aliquam sed vulputate justo, in ullamcorper leo. Ut nec augue non nunc accumsan pharetra. Nam volutpat feugiat ultrices. Quisque lacus diam, feugiat non maximus in, elementum quis tellus. Vivamus sed felis sit amet neque vulputate elementum vitae a mauris. Nulla suscipit, orci in lobortis hendrerit, dolor ipsum fermentum leo, et porttitor sem leo sit amet ex. In pellentesque tortor quis velit semper sodales. Nulla porttitor urna vel mauris rutrum, in feugiat est dapibus. In sed lorem mauris. Vestibulum sit amet porttitor nunc.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:amnesty-core/block-call-to-action {"preheading":"Give your readers an action at the end","title":"Add a call to action  ","content":"This can be a petition, a request for a donation, a call to join as an amnesty member or register for an online course. ","actionType":""} -->
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"className":"is-style-default"} -->
+<div class="wp-block-button is-style-default"><a class="wp-block-button__link wp-element-button">link here</a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons -->
+<!-- /wp:amnesty-core/block-call-to-action -->
```

Comment:
When previewing/inserting this pattern, I see a block validation error for the CTA:
![](http://bigbite.im/i/GqCC8c+)

Reply:
resolved